api/src/main/java/com/codeforcommunity/rest/ApiRouter.java

```diff
@@ -143,6 +144,18 @@ public Router initializeRouter(Vertx vertx) {
     Route getEventUsersRoute = router.get("/protected/eventcheckin/:id");
     getEventUsersRoute.handler(this::handleGetEventUsers);
 
+    Route createNewsRoute = router.post("/admin/news");
+    createEventRoute.handler(this::handleCreateNews);
+
+    Route getNewsRoute = router.get("/event/");
+    getEventRoute.handler(this::handleGetNews);
+
+    Route updateNewsRoute = router.put("/protected/event/:id");
+    updateEventRoute.handler(this::handleUpdateNews);
+
+    Route deleteNewsRoute = router.delete("/admin/event/:id");
+    deleteEventRoute.handler(this::handleDeleteNews);
+
```

Comment:
These routes dont quite match up. If only admins can create news then only admins should be able to update news as well. Maybe change update to an admin route?